app/src/config/env.ts

```diff
@@ -1,11 +1,13 @@
 export interface EnvVars {
     LAUNCHDARKLY_API_KEY: string;
     LAUNCHDARKLY_APP_NAME: string;
+    NODE_ENV: string;
 }
 
 const envVars: EnvVars = {
     LAUNCHDARKLY_API_KEY: process.env.LAUNCHDARKLY_API_KEY || '',
-    LAUNCHDARKLY_APP_NAME: process.env.LAUNCHDARKLY_APP_NAME || 'jenkins-for-jira'
+    LAUNCHDARKLY_APP_NAME: process.env.LAUNCHDARKLY_APP_NAME || 'jenkins-for-jira',
+    NODE_ENV: process.env.NODE_ENV || 'development'
```

Comment:
Trying to chase up Forge folks coz I'm running into some weird env things. With Forge, you can't use dotenv to get env variables coz Forge shits the bed. You have to set variables up as specified here https://developer.atlassian.com/platform/forge/environments-and-versions/. However, although the LD name and api key work just fine, the node env is coming through as 'production' even when I try to manually set it to development. The joys of Forge :sigh: